### PR TITLE
dotCMS/core#10400 popup menus styles

### PR DESCRIPTION
**Pull request by woontsoudev**
- popup menus styles

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

**Review by fmontes — CHANGES_REQUESTED** [review]
All the messures in line-height or paddings etc... are varibles, needs to confirm this.

**Review by jdotcms — APPROVED** [review]

File: scss/backend/dot-admin/components/_context-menu.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.site-browser__context-menus .site-browser__context-menu {
+    box-shadow: $menu-shadow;
+
+    a {
+        font-size: $font-size-medium;
+        height: $menu-item-height;
+        line-height: $menu-item-line-height;
+        padding: 4px 26px;
```
Comment (fmontes) [inline scss/backend/dot-admin/components/_context-menu.scss]: ```padding: ($basic-padding / 2) ($basic-padding * 3);```

But I don't see why we need so much left and right padding.

File: scss/backend/dot-admin/components/_context-menu.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.site-browser__context-menus .site-browser__context-menu {
+    box-shadow: $menu-shadow;
+
+    a {
+        font-size: $font-size-medium;
+        height: $menu-item-height;
+        line-height: $menu-item-line-height;
```
Comment (fmontes) [inline scss/backend/dot-admin/components/_context-menu.scss]: I don't see why we need a custom line-height variable, if I'm understanding correctly we should do:

```line-height: $menu-item-height;```

File: scss/_dijit-theme-dotcms.scss [hunk 3]
```diff
@@ -474,6 +474,9 @@ $menu-item-hover-bgcolor: $gray-ligthest;
 $menu-item-hover-textcolor: $gray-dark;
 $menu-item-selected-bgcolor: $menu-item-hover-bgcolor;
 $menu-item-selected-textcolor: $menu-item-hover-textcolor;
+$menu-item-height: $field-height;
+$menu-item-line-height: $field-height - 10;
```
Comment (fmontes) [inline scss/_dijit-theme-dotcms.scss]: I don't think we need this var. Also 10? where is that coming from?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**